public/usage-examples/graphics/text_width/text_width-1-simple.py

```diff
@@ -0,0 +1,25 @@
+from splashkit import *
+
+# Open a window
+open_window("Text Width", 800, 600)
+
+# Clear the screen to a black background
+clear_screen(color_white())
+
+# Text to display
+text = "Text Width!"
+
+# Calculate the text width with Normal font '0' and size 16
+text_width = text_width(text, 0, 16)
```

Comment:
There is a warning with the font which is affecting the text. I recommend loading a font, it would be best to download one from the internet instead of relying on the default font.